client/my-sites/plan-features/actions.jsx

```diff
@@ -50,6 +53,10 @@ const PlanFeaturesActions = ( {
 		if ( isLandingPage ) {
 			buttonText = translate( 'Select', { context: 'button' } );
 		}
+		if ( isInSignup && ( abtest( 'signupPlansCallToAction' ) === 'modified' ) ) {
+			buttonText = 'Start with ' + planName;
```

Comment:
@fditrapani Have a look at the [docs](https://github.com/Automattic/i18n-calypso/blob/master/README.md#translate-method), there are some examples that should help you :)